skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/api/Tracing.java

```diff
@@ -21,4 +21,35 @@ public static String getTraceId() {
 
         return formatTraceId(spanData.getTraceId());
     }
+
+
+    /**
+     * Get the latest span of current trace.
+     *
+     * @return span. if it exists, or null
+     */
+    public static Span getCurrentSpan() {
+        Span spanData = CurrentThreadSpanStack.peek();
+        return spanData;
+    }
+
+    /**
+     * Set the tag to span
+     *
+     * @param span
+     * @param tagKey key of tag
+     * @param tagValue value of tag
+     */
+    public static void tag(Span span, String tagKey, String tagValue) {
+        span.tag(tagKey, tagValue);
+    }
+
+    /**
+     * set the ref context
+     *
+     * @param contextData context data
+     */
+    public static void putRefContext(ContextData contextData){
```

Comment:
`putRefContext`'s parameter should be `RefContext` type. And caller in charge of converting `ContextData` to `RefContext`.

`putRefContext` should be named `set...` or `init...`. There is nothing to put here.